### PR TITLE
satellite/accounting: fix project accounting test

### PR DESCRIPTION
**Pull request by cam-a**
`TestProjectUsageStorage` was flaky with tally running: https://storjlabs.atlassian.net/browse/V3-2852. After investigating, I discovered the root of the issue. The first test case uploads 50 KiB and asserts no error (we have not exceeded project usage). The second test case manually inserts a bucket tally into the projectAccountingDB exceeding the storage limit, then asserts that we have exceeded the limit (this was the failure). 

If a tally iteration happens to finish _after_ the manual insert of bucket storage data, but _before_ the `ExceedsStorageUsage` check, the check will fail because tally will have inserted a new bucket storage tally from the pointer data of the 50 KiB upload, which is not exceeding the limit

To fix this, we just upload data rather than manually insert a bucket tally

Please describe the tests:
 - Test 1:
 - Test 2:
 
Please describe the performance impact:

## Code Review Checklist (to be filled out by reviewer)
 - [ ] NEW: Are there any Satellite database migrations? Are they forwards _and_ backwards compatible? 
 - [ ] Does the PR describe what changes are being made?
 - [ ] Does the PR describe why the changes are being made?
 - [ ] Does the code follow [our style guide](https://github.com/storj/docs/blob/master/code/Style.md)?
 - [ ] Does the code follow [our testing guide](https://github.com/storj/docs/blob/master/code/Testing.md)?
 - [ ] Is the PR appropriately sized? (If it could be broken into smaller PRs it should be)
 - [ ] Does the new code have enough tests? (*every* PR should have tests or justification otherwise. Bug-fix PRs especially)
 - [ ] Does the new code have enough documentation that answers "how do I use it?" and "what does it do?"? (both source documentation and [higher level](https://github.com/storj/docs), diagrams?)
 - [ ] Does any documentation need updating?
 - [ ] Do the database access patterns make sense?

### CODE REVIEW

**Comment by egonelbre** [comment]
Also retry running the tests with intervals set to really low.

**Comment by jenlij** [comment]
This is looking good, let me know when you add the test above and need an approval

**Comment by cam-a** [comment]
> Also retry running the tests with intervals set to really low.
> This is looking good, let me know when you add the test above and need an approval

I tested with tally interval to 1s and 1ns. However, the test usually only takes a little over a second to begin with, so I put a `TriggerWait` between the two uploads and it passes whereas the previous test fails with the expected message. Is this something we wanted to _add_ into the test, or just manually checking like I have?

**Comment by egonelbre** [comment]
I added it to the test, but it probably makes more sense to make a separate test to  ensure that once something has been uploaded `GetProjectStorageUsage` doesn't return `0` any more.

I currently added something really basic... upload some data and then start polling project usage service to see the values it returns. It should never return `0` again after the upload finishes.

**Comment by egonelbre** [comment]
Also when I lower `defaultInterval` in testplanet package to `300ms` I'm getting:
```
$ go test ./satellite/accounting
--- FAIL: TestStorageNodeUsage (0.00s)
    --- FAIL: TestStorageNodeUsage/Postgres/Postgres (0.68s)
        run.go:82: schema-suffix  681d0d86d1e9
        logger.go:130: 2019-11-11T16:46:46.684+0200     DEBUG   Connected to:   {"db source": "postgres://storj:storj-pass@localhost/teststorj?sslmode=disable&search_path=teststoragenodeusage%2Fpostgres%2Fpostgres%2F681d0d86d1e9%2Ft0&application_name=Satellite"}
        logger.go:130: 2019-11-11T16:46:46.728+0200     INFO    migrate Database Version        {"version": 65}
        --- FAIL: TestStorageNodeUsage/Postgres/Postgres/usage_with_pending_tallies (0.00s)
            db_test.go:108:
                        Error Trace:    db_test.go:108
                        Error:          Not equal:
                                        expected: time.Time{wall:0x0, ext:63708940800, loc:(*time.Location)(nil)}
                                        actual  : time.Time{wall:0x0, ext:63708933600, loc:(*time.Location)(nil)}

                                        Diff:
                                        --- Expected
                                        +++ Actual
                                        @@ -1,2 +1,2 @@
                                        -(time.Time) 2019-11-10 00:00:00 +0000 UTC
                                        +(time.Time) 2019-11-09 22:00:00 +0000 UTC

                        Test:           TestStorageNodeUsage/Postgres/Postgres/usage_with_pending_tallies
            db_test.go:109:
                        Error Trace:    db_test.go:109
                        Error:          Not equal:
                                        expected: 5934
                                        actual  : 11806
                        Test:           TestStorageNodeUsage/Postgres/Postgres/usage_with_pending_tallies
FAIL
FAIL    storj.io/storj/satellite/accounting     19.859s
FAIL
```

File: satellite/accounting/projectusage.go [hunk 1]
```diff
@@ -124,7 +124,8 @@ func (usage *ProjectUsage) ExceedsStorageUsage(ctx context.Context, projectID uu
 	return false, limit, nil
 }
 
-func (usage *ProjectUsage) getProjectStorageTotals(ctx context.Context, projectID uuid.UUID) (total int64, err error) {
+// GetProjectStorageTotals retursn how much storage has been used in total.
```
Comment (JessicaGreben) [inline satellite/accounting/projectusage.go]: retursn -> returns

**Comment by cam-a** [comment]
moved to https://review.dev.storj.io/c/storj/storj/+/291

**Pull request closed without merging**